subgraphs/axelar/src/mappings.ts

```diff
@@ -0,0 +1,762 @@
+
+  const customEvent = _createCustomEvent(event)!;
+  const account = _handleTransferIn(
+    Address.fromString(tokenAddress),
+    srcAccount,
+    event.params.contractAddress,
+    event.params.amount,
+    srcChainId,
+    srcPoolId,
+    poolId,
+    BridgePoolType.BURN_MINT,
+    CrosschainTokenType.CANONICAL,
+    customEvent,
+    event.params.commandId
+  );
+
+  account.messageIn(srcChainId, srcAccount, event.params.payloadHash);
+}
+
+export function handleCommandExecuted(event: Executed): void {
+  log.info("[handleCommandExecuted]commandId={} tx {} logIndex={}", [
+    event.params.commandId.toHexString(),
+    event.transaction.hash.toHexString(),
+    event.transactionLogIndex.toString(),
+  ]);
+
+  // command is one of deployToken, mintToken, approveContractCall, approveContractCallWithMint,
+  // burnToken or transferOperatorship
+  // https://github.com/axelarnetwork/axelar-cgp-solidity/blob/2a24602fdad6d3aa80f4e43cacfe7241adbb905e/contracts/AxelarGateway.sol#L308-L319
+  // We only interest in mintToken and burnToken, we detect those events
+  // by searching the log receipt for Transfer event
+  const receipt = event.receipt;
+  if (!receipt) {
+    log.error("[handleCommandExecuted]No receipt for tx {}", [
+      event.transaction.hash.toHexString(),
+    ]);
+    return;
+  }
+
+  const transferSignature = crypto.keccak256(
+    ByteArray.fromUTF8("Transfer(address,address,uint256)")
+  );
+
+  const commandId = event.params.commandId;
+  let command = _Command.load(commandId);
+  if (!command) {
+    command = new _Command(commandId);
+    command.isBurnToken = false;
+    command.isProcessed = false;
+    command.save();
+  }
+
+  const logs = event.receipt!.logs;
+  //Transfer has a logIndex that's one below event.logIndex
+  const transferLogIndex = event.logIndex.minus(BIGINT_ONE);
+  for (let i = 0; i < logs.length; i++) {
+    const thisLog = logs[i];
+    if (thisLog.logIndex.gt(transferLogIndex)) {
+      return;
+    }
+    // topics[0] - signature
+    // topics[1] - from address
+    // topics[2] - to address
+    const logSignature = thisLog.topics[0];
+    if (
+      transferLogIndex.equals(thisLog.logIndex) &&
+      logSignature == transferSignature
+    ) {
+      const tokenAddress = thisLog.address;
+      const fromAddress = ethereum
+        .decode("address", thisLog.topics[1])!
+        .toAddress();
+      const toAddress = ethereum
+        .decode("address", thisLog.topics[2])!
+        .toAddress();
+      const transferAmount = ethereum
+        .decode("uint256", thisLog.data)!
+        .toBigInt();
+
+      log.info(
+        "[handleCommandExecuted]from={} to={} thisLog.data={} tx={}-{}",
+        [
+          fromAddress.toHexString(),
+          toAddress.toHexString(),
+          thisLog.data.toHexString(),
+          thisLog.transactionHash.toHexString(),
+          thisLog.logIndex.toString(),
+        ]
+      );
+
+      log.info("[handleCommandExecuted]to==0x0 {} tx={}-{}", [
```

Comment:
why do we need this log? It seems to have the same informationas the one above but the to address is 0, which you could see in the above log

Reply:
good point. it is remain from debugging. deleted.